### PR TITLE
Limit newly created lanes to BOOK_MEDIUM

### PR DESCRIPTION
**Pull request by leonardr**
This branch brings in recent submodule updates, and also makes sure that when you create a lane through the admin interface it only contains ebooks.

### CODE REVIEW

File: bin/opds_entry_coverage [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
```
Comment (leonardr) [inline bin/opds_entry_coverage]: See https://github.com/NYPL-Simplified/circulation-docker/pull/73 for this.

Comment (aslagle) [reply bin/opds_entry_coverage]: Maybe you could make this a symlink to the script in core/bin and then you wouldn't have to duplicate it.

Comment (leonardr) [reply bin/opds_entry_coverage]: I don't think a symlink would work because the import also needs to change.

Comment (aslagle) [reply bin/opds_entry_coverage]: I'd have to test it but I think the core script would import relative to itself regardless of where you ran it from.

Comment (leonardr) [reply bin/opds_entry_coverage]: I'll try it out.

Comment (leonardr) [reply bin/opds_entry_coverage]: No, it doesn't work. :(

```
simplified/circulation$ bin/opds_entry_coverage
Traceback (most recent call last):
  File "bin/opds_entry_coverage", line 8, in <module>
    from coverage import OPDSEntryWorkCoverageProvider
ImportError: No module named coverage
```

**Pull request merged**